tox.ini

```diff
@@ -29,3 +29,12 @@ description = Build documentation
 deps = -r{toxinidir}/docs/requirements.txt
 commands =
     sphinx-build -T -E -W --keep-going {tty:--color} -j auto -d docs/build/doctrees -b html docs docs/build/html
+
+[testenv:clean]
+description = Erase docs and coverage artifacts
+deps =
+skip_install = True
+allow_external = /bin/sh
+commands =
+    /bin/sh -c "rm -rf {toxinidir}/test/coverage/*"
```

Comment:
@samdoran I have a few comments here:
1) You could've used a combination of `git clean` commands which is easier to maintain, here's an example: https://github.com/ansible/pylibssh/blob/924771a/tox.ini#L369-L380.
2) Instead of calling sh/rm, you could do `{envpython} -c 'import shutil; shutil.rmdir("{toxinidir}/test/coverage/")'` and avoid having to use `allow_externals`.